packages/components/pie-cookie-banner/README.md

```diff
@@ -11,17 +11,53 @@
 # Table of Contents
 
 1. [Introduction](#pie-cookie-banner)
-2. [Local Development](#local-development)
+2. [Installation](#installation)
 3. [Importing the component](#importing-the-component)
-4. [Props](#props)
-5. [Testing](#testing)
+4. [Peer Dependencies](#peer-dependencies)
+5. [Local Development](#local-development)
+6. [Props](#props)
+7. [Testing](#testing)
```

Comment:
I think we'll need an events section for the cookie banner, whether it's part of this PR or another ticket

Reply:
I'll add that as a todo and create it as part of the cookie banner work 👍